service.unmanic/settings-default.xml

```diff
@@ -0,0 +1,3 @@
+<settings>
```

Comment:
Why this file?

Reply:
Left over from previous build. 
Sorry. Will remove...

Reply:
Well, the thing is that Kodi can also cause the abort when exiting. In this case, your service will then also restart, which is not what should happen.